src/Commands/UpdateEditorLinks.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Drupal\tide_core\Commands;
+
+use Drush\Commands\DrushCommands;
+
+/**
+ * A drush command file.
+ *
+ * @package Drupal\tide_core\Commands
+ */
+class UpdateEditorLinks extends DrushCommands {
+
+  /**
+   * Install anchor_link module and add settings.
+   *
+   * @command tide-core:update-editor-links
+   * @aliases update-editor-links
+   * @usage tide-core:update-editor-links
+   */
+  public function update() {
```

Comment:
@henrytranvan this entire custom Drush Command should be a `hook_update_N` implementation. Nobody will know to run a Drush command when updating this module, so that they can have the CKEditor link features you've configured. 

Take a look at other existing `hook_update_N`  implementations used in tide modules, where they import config files. An example is in 

https://github.com/dpc-sdp/tide_site/pull/98/files#diff-880fa7b656cf6dbc0899497b2e7fb9b933847200fb1d3fd97a7e13d610dd3100R428-R475 

I can help you create something similar if you need assistance.